Failed to resolve 'com.gwtplatform.mvp.client.ApplicationController' via deferred binding

i switched today to 0.8.4 and want to use the new MvpWithEntryPoint and followed the wiki entry.
but when it now comes to gwt compile i got these exception:
`
[INFO] --- gwt-maven-plugin:2.5.0:compile (default) @ aqm ---
[INFO] Compiling module de.bigmichi1.aqm.Main
[INFO]    Computing all possible rebind results for 'com.gwtplatform.mvp.client.ApplicationController'
[INFO]       Rebinding com.gwtplatform.mvp.client.ApplicationController
[INFO]          Invoking generator com.gwtplatform.mvp.rebind.ApplicationControllerGenerator
[INFO]             [ERROR] Generator 'com.gwtplatform.mvp.rebind.ApplicationControllerGenerator' threw an exception while rebinding 'com.gwtplatform.mvp.client.ApplicationController'
[INFO] java.lang.NullPointerException
[INFO]  at com.google.gwt.user.rebind.ClassSourceFileComposer.print(ClassSourceFileComposer.java:157)
[INFO]  at com.google.gwt.user.rebind.ClassSourceFileComposer.println(ClassSourceFileComposer.java:176)
[INFO]  at com.google.gwt.user.rebind.ClassSourceFileComposer.<init>(ClassSourceFileComposer.java:63)
[INFO]  at com.google.gwt.user.rebind.ClassSourceFileComposerFactory.createSourceWriter(ClassSourceFileComposerFactory.java:106)
[INFO]  at com.gwtplatform.mvp.rebind.ApplicationControllerGenerator.generate(ApplicationControllerGenerator.java:71)
[INFO]  at com.google.gwt.core.ext.IncrementalGenerator.generateNonIncrementally(IncrementalGenerator.java:40)
[INFO]  at com.google.gwt.dev.javac.StandardGeneratorContext.runGeneratorIncrementally(StandardGeneratorContext.java:657)
[INFO]  at com.google.gwt.dev.cfg.RuleGenerateWith.realize(RuleGenerateWith.java:41)
[INFO]  at com.google.gwt.dev.shell.StandardRebindOracle$Rebinder.rebind(StandardRebindOracle.java:79)
[INFO]  at com.google.gwt.dev.shell.StandardRebindOracle.rebind(StandardRebindOracle.java:276)
[INFO]  at com.google.gwt.dev.shell.StandardRebindOracle.rebind(StandardRebindOracle.java:265)
[INFO]  at com.google.gwt.dev.DistillerRebindPermutationOracle.getAllPossibleRebindAnswers(DistillerRebindPermutationOracle.java:91)
[INFO]  at com.google.gwt.dev.jjs.impl.UnifyAst$UnifyVisitor.handleGwtCreate(UnifyAst.java:355)
[INFO]  at com.google.gwt.dev.jjs.impl.UnifyAst$UnifyVisitor.handleMagicMethodCall(UnifyAst.java:433)
[INFO]  at com.google.gwt.dev.jjs.impl.UnifyAst$UnifyVisitor.endVisit(UnifyAst.java:237)
[INFO]  at com.google.gwt.dev.jjs.ast.JMethodCall.traverse(JMethodCall.java:243)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.traverse(JModVisitor.java:361)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.accept(JModVisitor.java:273)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.accept(JModVisitor.java:265)
[INFO]  at com.google.gwt.dev.jjs.ast.JVisitor.accept(JVisitor.java:116)
[INFO]  at com.google.gwt.dev.jjs.ast.JCastOperation.traverse(JCastOperation.java:65)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.traverse(JModVisitor.java:361)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.accept(JModVisitor.java:273)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.accept(JModVisitor.java:265)
[INFO]  at com.google.gwt.dev.jjs.ast.JVisitor.accept(JVisitor.java:116)
[INFO]  at com.google.gwt.dev.jjs.ast.JDeclarationStatement.traverse(JDeclarationStatement.java:48)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor$ListContextImmutable.traverse(JModVisitor.java:170)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.acceptWithInsertRemoveImmutable(JModVisitor.java:336)
[INFO]  at com.google.gwt.dev.jjs.ast.JBlock.traverse(JBlock.java:83)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.traverse(JModVisitor.java:361)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.accept(JModVisitor.java:273)
[INFO]  at com.google.gwt.dev.jjs.ast.JVisitor.accept(JVisitor.java:137)
[INFO]  at com.google.gwt.dev.jjs.ast.JVisitor.accept(JVisitor.java:133)
[INFO]  at com.google.gwt.dev.jjs.ast.JMethodBody.traverse(JMethodBody.java:82)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.traverse(JModVisitor.java:361)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.accept(JModVisitor.java:273)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.accept(JModVisitor.java:265)
[INFO]  at com.google.gwt.dev.jjs.ast.JMethod.visitChildren(JMethod.java:426)
[INFO]  at com.google.gwt.dev.jjs.ast.JMethod.traverse(JMethod.java:395)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.traverse(JModVisitor.java:361)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.accept(JModVisitor.java:273)
[INFO]  at com.google.gwt.dev.jjs.ast.JModVisitor.accept(JModVisitor.java:265)
[INFO]  at com.google.gwt.dev.jjs.impl.UnifyAst.mainLoop(UnifyAst.java:902)
[INFO]  at com.google.gwt.dev.jjs.impl.UnifyAst.exec(UnifyAst.java:627)
[INFO]  at com.google.gwt.dev.jjs.JavaToJavaScriptCompiler.precompile(JavaToJavaScriptCompiler.java:624)
[INFO]  at com.google.gwt.dev.jjs.JavaScriptCompiler.precompile(JavaScriptCompiler.java:33)
[INFO]  at com.google.gwt.dev.Precompile.precompile(Precompile.java:278)
[INFO]  at com.google.gwt.dev.Precompile.precompile(Precompile.java:229)
[INFO]  at com.google.gwt.dev.Precompile.precompile(Precompile.java:141)
[INFO]  at com.google.gwt.dev.Compiler.run(Compiler.java:232)
[INFO]  at com.google.gwt.dev.Compiler.run(Compiler.java:198)
[INFO]  at com.google.gwt.dev.Compiler$1.run(Compiler.java:170)
[INFO]  at com.google.gwt.dev.CompileTaskRunner.doRun(CompileTaskRunner.java:88)
[INFO]  at com.google.gwt.dev.CompileTaskRunner.runWithAppropriateLogger(CompileTaskRunner.java:82)
[INFO]  at com.google.gwt.dev.Compiler.main(Compiler.java:177)
[INFO]    [ERROR] Errors in 'com/gwtplatform/mvp/client/ClientEntryPoint.java'
[INFO]       [ERROR] Line 23: Failed to resolve 'com.gwtplatform.mvp.client.ApplicationController' via deferred binding
`
i have no idea whats going wrong here

--- a/gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/rebind/ApplicationControllerGenerator.java
+++ b/gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/rebind/ApplicationControllerGenerator.java
@@ -65,6 +65,10 @@ public class ApplicationControllerGenerator extends AbstractGenerator {
 
         PrintWriter printWriter = tryCreatePrintWriter(generatorContext, SUFFIX);
 
+        if (printWriter == null) {
+            return typeName + SUFFIX;
+        }
+
         JClassType preBootstrapper = getPreBootstrapper();
 
         ClassSourceFileComposerFactory composer = initComposer(preBootstrapper);

